modules/coinswap/keeper/migrations.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	v150 "github.com/irisnet/irismod/modules/coinswap/migrations/v150"
 	v152 "github.com/irisnet/irismod/modules/coinswap/migrations/v152"
+	v153 "github.com/irisnet/irismod/modules/coinswap/migrations/v153"
 )
```

Comment:
rename package name `v150`,`v152`.`v153` to `v1`,`v2`,'v3', consistent with moduleVersion